sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -487,6 +487,10 @@ object DDLUtils {
     isDatasourceTable(table.properties)
   }
 
+  def isSchemaInferred(table: CatalogTable): Boolean = {
+    table.properties.get(DATASOURCE_SCHEMA_TYPE) == Option(SchemaType.INFERRED.name)
```

Comment:
Please don't use contains. It makes it much harder to read and understand
the return type is an option.

On Sunday, July 17, 2016, Jacek Laskowski <notifications@github.com> wrote:

> In
> sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala
> <https://github.com/apache/spark/pull/14207#discussion_r71083304>:
>
> > @@ -487,6 +487,10 @@ object DDLUtils {
> >      isDatasourceTable(table.properties)
> >    }
> >
> > +  def isSchemaInferred(table: CatalogTable): Boolean = {
> > +    table.properties.get(DATASOURCE_SCHEMA_TYPE) == Option(SchemaType.INFERRED.name)
>
> Consider contains.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/spark/pull/14207/files/3be0dc0b7cfd942459c598c0d35f3d67a2c020ba#r71083304>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AATvPLJxrOTgsryjrhIAFMb3v7t5vl8-ks5qWjylgaJpZM4JMzdl>
> .
>